doc/MANUAL.txt

```diff
@@ -41,6 +75,10 @@ options that can be set within it::
 
   $ testr help run
 
+Currently all the configuration pertains to the (optional) testrepository
+runner. The `load`, `failing`, `last` and other query commands work on the
+data store don't have configuration.
```

Comment:
Maybe I'm just tired, but these two sentences seem hard to understand. How about something like:

```
Currently all the configuration options pertain to the optional testrepository runner. The query commands (`load`, `failing`, `last`, for example) work on the data store, and do not have any configuration options.
```